.coveragerc

```diff
@@ -13,3 +13,4 @@ omit =
   *app/__init__.py             
   *app/auth/__init__.py
   *app/mail.py
+  *app/databases/*
```

Comment:
The `utilities.py` file is a testable functionality which should not be excluded

Reply:
Noted. Thanks